### PR TITLE
Add additional Hippa Hitrust v9.2 compliance queries enhancement Closes #54

### LINKED ISSUE #54: Add additional HIPAA HITRUST 9.2  compliance queries
**Is your feature request related to a problem? Please describe.**
Add Hippa Hitrust v9.2 compliance queries

**Describe the solution you'd like**
Queries to achieve the Hippa Hitrust compliance to be added.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by krishna5891**
### Checklist
- [x] Issue(s) linked

### CODE REVIEW

**Review by rajlearner17 — APPROVED** [review]
LGTM

File: query/appservice/appservice_function_app_only_https_accessible.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
+select
+  -- Required Column
+  app.id as resource,
+  case
+    when not https_only then 'ok'
```
Comment (khushboo9024) [inline query/appservice/appservice_function_app_only_https_accessible.sql:5]: This case should throw alarm, please check once.
`Function App should only be accessible over HTTPS`

Comment (krishna5891) [reply query/appservice/appservice_function_app_only_https_accessible.sql]: updated the changes.

**Comment by krishna5891** [comment]
all these files have been updated in #53. Closing this as duplicate PR.

**Pull request closed without merging**